app/src/main/java/de/tum/in/tumcampusapp/utils/Const.kt

```diff
@@ -131,6 +131,7 @@ object Const {
     const val EVENT_START = "pVon"
     const val EVENT_END = "pBis"
     const val EVENT_NR = "pTerminNr"
+    const val EVENT_DEFAULT_DURATION = 7200000 // Milliseconds
```

Comment:
Please don't do this.

Reply:
I moved that to the companion object of the Event class.